paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -78,47 +78,93 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
     private val userFacingLogger: UserFacingLogger,
 ) : PaymentSheetLoader {
 
+    @Suppress("LongMethod")
     override suspend fun load(
         initializationMode: PaymentSheet.InitializationMode,
         paymentSheetConfiguration: PaymentSheet.Configuration,
         isReloadingAfterProcessDeath: Boolean,
         initializedViaCompose: Boolean,
     ): Result<PaymentSheetState.Full> = withContext(workContext) {
-        eventReporter.onLoadStarted(initializedViaCompose)
+        runCatching {
```

Comment:
can we write a help that wraps these 3 things (withContext, runCatching, coroutineScope) so we don't need so much nesting?

Reply:
The we could leave a comment similar to what you wrote in github here to make it clear to the readers why it's needed.

Reply:
Maybe `withContextCatching`